workers/manager-Dockerfile

```diff
@@ -1,7 +1,11 @@
-FROM python:3.8-buster
+FROM python:3.12-slim-bookworm
 LABEL zimfarm=true
 LABEL org.opencontainers.image.source https://github.com/openzim/zimfarm
 
+# install libmagic for zimscraperlib
+RUN apt-get update -y \
+    && apt-get install -y --no-install-recommends libmagic1
```

Comment:
Why would we want scraperlib in manager??

Reply:
Copy-paste issue, my bad.